### PR TITLE
gtag after consent

### PR DESCRIPTION
**Pull request by sbatten**

### CODE REVIEW

File: js/page.js [hunk 1]
```diff
@@ -4,10 +4,30 @@ $('#small-nav-dropdown').change(function() {
...
 
+const site_tag = 'UA-62780441-30';
+function loadAnalytics(gtag) {
+  // set cookie to expire in 12 x 28 days
+  gtag('config', site_tag, { 'anonymize_ip': true, 'cookie_expires': 29030400 })
+}
+
 function onConsentChanged() {
-  console.log(WcpConsent.siteConsent.getConsent());
+  if (!consentRequired()) {
```
Comment (gregvanl) [inline js/page.js:14]: if consent isn't required, don't want to create the UA-62780441-30 gtag?

Comment (sbatten) [reply js/page.js]: fixed

Comment (gregvanl) [reply js/page.js]: We'll need the same change in the LSP site PR.

Comment (sbatten) [reply js/page.js]: it already got merged before I saw this comment so I just updated directly

Comment (gregvanl) [reply js/page.js]: Did I merge the LSP fix by accident?

Comment (gregvanl) [reply js/page.js]: Oh, I see Dirk did. Thanks for fixing these. I'll kick off the WCP scans again to make sure we're green again.

**Pull request merged**